src/01-Learning-JS/08-load-data.md

```diff
@@ -216,11 +216,17 @@ If you wish to read about them, see Observable Framework's "[Data loaders](https
 
 ### E1. Understanding CSV file formats
 
-Goal: Demonstrate your understanding of the CSV file format by creating a mini CSV file with **3 columns and 5 rows with the topic of your choice**.
+Goal: Demonstrate your understanding of the CSV file format b creating a mini CSV file with **3 columns and 5 rows with the topic of your choice**.
 
 Use the empty codeblock below.
 
 ```
+name, age, job;
```

Comment:
Use no spaces and each new row is delimited by a "return carriage" aka a new line

```suggestion
name,age,job
```